README.md

```diff
@@ -237,3 +237,42 @@ def attacker_factory():
         }
     }
 ```
+
+### Allowlisting and denylisting Swagger tags and operations
+
+We can use fixtures to control whether fuzz-lightyear fuzzes certain parts of
+the Swagger specification. This allows developers to only fuzz the parts
+of the specification that can be fuzzed in the test environment.
+
+```python
+import fuzz_lightyear
+
+
+@fuzz_lightyear.include.tags
+def get_tags_to_fuzz():
+    """fuzz_lightyear will only fuzz operations from
+    these tags.
+    """
+    return ['user', 'transactions']
+
+
+@fuzz_lightyear.exclude.operations
+def get_operations_to_exclude():
+    """fuzz_lightyear will not call these Swagger
+    operations.
+    """
+    return [
+        'get_user_id',
+        'operation_doesnt_work_in_test_environment',
+    ]
+
+
+@fuzz_lightyear.exclude.non_vulnerable_operations
+def get_non_vulnerable_operations():
+    """fuzz_lightyear will not check these Swagger
+    operations for vulnerabilities.
+    """
+    # Accessing a user's public profile shouldn't require
+    # authentication.
+    return ['get_user_public_profile']
+```
```

Comment:
Can we write something about the difference between `exclude.non_vulnerable_operations` and `exclude.operations`?